satellite/satellitedb/dbx/gen.go

```diff
@@ -28,19 +28,21 @@ func init() {
 }
 
 // Unwrap returns the underlying error.
-func (e *Error) Unwrap() error {
-	return e.Err
-}
+func (e *Error) Unwrap() error { return e.Err }
+
+// Cause returns the underlying error.
+func (e *Error) Cause() error { return e.Err }
```

Comment:
Whats the difference to Unwrap?

Reply:
`zeebo/errs` uses `Cause` for unwrapping, Go 1.13 errors proposal uses `Unwrap`

Reply:
Okay, so we remove that once 1.13 is out?
Then its fine!

Reply:
We can remove it once we start using the new errors package.